api/test/unit/validation/validationMiddleware.test.js

```diff
@@ -365,4 +365,209 @@ describe('Validate Policy JSON Schema structure', function() {
       done();
     });
   });
+  it('should successfully validate policy schema without start_date and end_date in recurring_schedule', function(done) {
```

Comment:
Change any of them. I changed a 400 to a 200, as well as false to true, and npm test just passed.